### PR TITLE
Normalize addresses to lowercase

### PR DESCRIPTION
**Pull request by xianny**
@PirosB3 pointed out that retrieving orders from `GET /sra/v3/orders` doesn't work if the maker address is not all lowercase.

On further investigation I found that retrieving orders required exact string matches between query params and postgres records.

This PR normalizes addresses from incoming requests to all lowercase, so that API requests should be case-insensitive to addresses. Normalization is implemented across all endpoints for consistency.

### CODE REVIEW

**Comment by xianny** [comment]
Moved address normalization into a middleware function.

File: src/services/orderbook_service.ts [hunk 1]
```diff
@@ -71,10 +71,10 @@ export class OrderBookService {
     ): Promise<OrderbookResponse> {
         const [bidSignedOrderEntities, askSignedOrderEntities] = await Promise.all([
             this._connection.manager.find(SignedOrderEntity, {
-                where: { takerAssetData: baseAssetData, makerAssetData: quoteAssetData },
+                where: { takerAssetData: baseAssetData.toLowerCase(), makerAssetData: quoteAssetData.toLowerCase() },
```
Comment (fragosti) [inline src/services/orderbook_service.ts:74]: Do we still need these calls?

File: test/app_test.ts [hunk 2]
```diff
@@ -54,4 +57,31 @@ describe('app test', () => {
...
+            .expect(HttpStatus.OK)
+            .then(response => {
+                expect(response.body.perPage).to.equal(DEFAULT_PER_PAGE);
+                expect(response.body.page).to.equal(DEFAULT_PAGE);
+                expect(response.body.total).to.equal(1);
+                expect(response.body.records[0].order).to.deep.equal(apiOrderResponse);
+            });
+
+        await dependencies.connection.manager.remove(orderModel);
+    });
```
Comment (fragosti) [inline test/app_test.ts:86]: Noice

File: src/runners/http_service_runner.ts [hunk 3]
```diff
@@ -60,6 +61,9 @@ export async function runHttpServiceAsync(
     server.keepAliveTimeout = config.HTTP_KEEP_ALIVE_TIMEOUT;
     server.headersTimeout = config.HTTP_HEADERS_TIMEOUT;
 
+    // transform all values of `req.query.[xx]Address` to lowercase
+    app.use(addressNormalizer);
```
Comment (fragosti) [inline src/runners/http_service_runner.ts:65]: We need to add this to the monolith runner too right?

**Comment by xianny** [comment]
deploy staging

**Comment by xianny** [comment]
deploy staging

**Pull request merged**